src/components/OpenTofuNotice.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import Admonition from "@theme/Admonition"
+
+export const OpenTofuNotice = () => {
+    return (
+        <div>
+            <Admonition type="tip" title="OpenTofu Support">
+                <p>Gruntwork Library will support OpenTofu when OpenTofu becomes available.</p>
+
+                <p>To learn more about OpenTofu see the official <a href="https://opentofu.org/">OpenTofu website</a> and the current <a href="https://github.com/opentofu/opentofu/blob/main/WEEKLY_UPDATES.md">project status</a>.</p>
```

Comment:
Suggest slightly less verbose wording and single paragraph format:
```suggestion
                <p>Gruntwork Library will support OpenTofu when it becomes available. To learn more, see the official <a href="https://opentofu.org/">OpenTofu website</a> and <a href="https://github.com/opentofu/opentofu/blob/main/WEEKLY_UPDATES.md">project status</a>.</p>
```